README.md

```diff
@@ -17,35 +14,41 @@ This SDK is built on the AWS Common Runtime, a collection of libraries
 cross-platform, high-performance, secure, and reliable. The libraries are bound
 to Java by the [aws-crt-java](https://github.com/awslabs/aws-crt-java) package.
 
-Integration with AWS IoT Services such as
-[Device Shadow](https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html)
-[Jobs](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html)
-[Fleet Provisioning](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html)
-is provided by code that been generated from a model of the service.
 
-# Installation
-## Minimum Requirements
+*__Jump To:__*
+* [Installation](#Installation)
+* [Samples](samples)
+* [Getting Help](#Getting-Help)
+* [Giving Feedback and Contributions](#Giving-Feedback-and-Contributions)
+* [More Resources](#More-Resources)
+
+
+
+## Installation
+
+### Minimum Requirements
 *   Java 8 or above
 
 ** Set JAVA_HOME first
 
 *   Install Maven
-
+```
```

Comment:
This is mac-specific.  I think we should remove it and if someone needs help installing maven they can use the official documentation.

Reply:
removed